website/src/css/custom.scss

```diff
@@ -279,9 +279,9 @@ h1[class^="h1Heading"] {
   line-height: 36px;
   @include desktop {
     width: 48px;
-    span {
-      display: none;
-    }
+    text-indent: 9999px;
```

Comment:
What does this do?

Reply:
Because we've bump up docusaurus theme, some css changes has been made in their side. The text indent hide discord and github text from those logo.

Before:
<img width="814" alt="Capture d’écran 2022-03-31 à 19 48 18" src="https://user-images.githubusercontent.com/1186424/161118344-19455201-ccaa-4401-935f-30e6b1b56a0c.png">

After:
<img width="816" alt="Capture d’écran 2022-03-31 à 19 48 30" src="https://user-images.githubusercontent.com/1186424/161118383-6f6fb097-1629-46ca-b7b0-0f9e593b132b.png">

Reply:
That makes sense, let's push this through!